packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -110,6 +111,11 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
     return { ...this.defaultBIP44Params, accountNumber }
   }
 
+  getAddressNList(accountNumber: number): BIP32Path {
+    const p = this.getBIP44Params({ accountNumber })
+    return bip32ToAddressNList(`m/${p.purpose}'/${p.coinType}'/${accountNumber}'/0/0`)
+  }
+
```

Comment:
This essentially exists already as `bip32ToAddressNList(toPath(this.getBIP44Params({ accountNumber })))`

if anything, I feel like we can add an additional utility that encapsulates this in `utils/bip44.ts` -> `toAddressNList`/`fromAddressNList` taking in or returning `BIP44Params`